pkg/plugins/golang/deploy-image/v1alpha1/scaffolds/internal/templates/controllers/controller.go

```diff
@@ -0,0 +1,231 @@
+/*
```

Comment:
We will also need to scaffold the api file such as we do in the go/v3: https://github.com/kubernetes-sigs/kubebuilder/blob/master. So that we can add the spec size there too. 

We need to add the dir api at the same level as the controller
and copy the types.go for it.

See that we will remove the bool force https://github.com/kubernetes-sigs/kubebuilder/blob/master/pkg/plugins/golang/v3/scaffolds/internal/templates/api/types.go#L36
and we will have only the option: https://github.com/kubernetes-sigs/kubebuilder/blob/master/pkg/plugins/golang/v3/scaffolds/internal/templates/api/types.go#L58

So that we can Overwrite the api file with this plugin (not that we always call the api from the go/v3 and then this implementation so the api file from go will be scaffolded and our goal here is to replace/overwrite that by this plugin implementation)

Then, note that we will change the content of the boilerplate here: https://github.com/kubernetes-sigs/kubebuilder/blob/master/pkg/plugins/golang/v3/scaffolds/internal/templates/api/types.go#L82-L83

So that we can scaffold the size: https://github.com/operator-framework/operator-sdk/blob/master/testdata/go/v3/memcached-operator/api/v1alpha1/memcached_types.go#L33